src/coreimage.cs

```diff
@@ -419,7 +419,7 @@ interface CIContext_ImageRepresentation {
 		NSData GetJpegRepresentation (CIImage image, CGColorSpace colorSpace, CIImageRepresentationOptions options);
 #endif
 
-		[iOS (11,0)][TV (11,0)][Mac (10,13)]
+		[iOS (11,0)][TV (11,0)][Mac (10,13,4)]
```

Comment:
do we have any evidence it's correct ?
I trust our tests for availability (at least introduction) much more than apple's headers

Reply:
@spouliot mmm wouldn't this match the case where it might be available but its use restricted to >=10,13,4? just curious about your thoughts on this.

Reply:
I actually did not look at the diff - and yes it looks like it was not (at least publicly) available. Let's use the updated one (since we should not have made them public earlier).